spec/models/ext_management_system_spec.rb

```diff
@@ -466,8 +462,8 @@
       it "queues up destroy for child_managers" do
         described_class.destroy_queue(ems.id)
 
-        expect(MiqQueue.count).to eq(2)
-        expect(MiqQueue.pluck(:instance_id)).to include(ems.id, child_manager.id)
+        expect(MiqQueue.count).to eq(1)
```

Comment:
Again, the description needs to be updated since the child manager destroy is no longer queued.

Reply:
Now I've done some re-arrangement of the specs